src/test/run-pass/shift-near-oflo.rs

```diff
@@ -13,9 +13,6 @@
 // Check that we do *not* overflow on a number of edge cases.
 // (compare with test/run-fail/overflowing-{lsh,rsh}*.rs)
 
-// (Work around constant-evaluation)
-fn id<T>(x: T) -> T { x }
```

Comment:
(I'm not sure if this was a good idea. The point of this test was to exercise the code generated for execution at runtime, but with this change, you are heavily encouraging it to be evaluated at compile time.  I would probably have preferred to just leave this as it was, perhaps even modifying the `fn id` further to try to ensure that it would not be flagged as a const fn by future aggressive analyses...)